src/main/ipc.ts

```diff
@@ -41,6 +41,10 @@ ipcMain.on(IpcEvents.GET_VERSION, e => {
     e.returnValue = app.getVersion();
 });
 
+ipcMain.on(IpcEvents.GET_PLATFORM, e => {
+    e.returnValue = process.platform;
```

Comment:
this doesn't seem necessary. In preload you can just access it directly, in rendered you can use navigator.platform